python/pyspark/sql/functions.py

```diff
@@ -809,6 +809,36 @@ def ntile(n):
     return Column(sc._jvm.functions.ntile(int(n)))
 
 
+@since(2.3)
+def unboundedPreceding():
```

Comment:
Wait .. why do we this naming convention here? I thought we decided to stick to this_namimg_convention. Seems Scala side has this one which seems added in 2.3 -

https://github.com/apache/spark/blob/d5861aba9d80ca15ad3f22793b79822e470d6913/sql/core/src/main/scala/org/apache/spark/sql/functions.scala#L786-L788

@cloud-fan and @rxin, was this just a mistake or did I miss something?

Reply:
Oh! I am so sorry. I meant https://issues.apache.org/jira/browse/SPARK-10621 and this one seems a different case. Please ignore my comment above.